tests/integration/solidity_contracts/PlainOpcodes/Safe.sol

```diff
@@ -2,6 +2,9 @@
 pragma solidity ^0.8.13;
 
 contract Safe {
+    constructor() payable{
+    }
```

Comment:
sol lint would make this `constructor() payable {}` you should add it to your vscode